backend/services/applications.js

```diff
@@ -178,7 +178,16 @@ async function createApplication(raw_application) {
  * Returns a list of JSON objects representing all applications in the system.
  * Can be filtered using options (e.g. to fetch only open applications, etc.)
  */
-async function getAllApplications(start_date, end_date) {
+async function getAllApplications(start_date, end_date, final = null) {
```

Comment:
Wait I was talking about this part how we have final = null. I feel like it makes sense to be final = false.

Reply:
Ohh I see, yeah I see that I just fixed it